apps/plato_single_variant/Readme.md

```diff
@@ -1,20 +1,17 @@
-<!-- dx-header -->
 # PLATO Single Variant Analysis
-<!-- /dx-header -->
 
 ## What does this app do?
+This app performs genetic associations using PLATO (the PLatform for the Analysis, Translation and Organization of large-scale data). PLATO is a software tool equipped to handle whole-genome sequence data for hundreds of thousands of samples to explore complexity and genetic architecture underlying common traits using phenome-wide association studies (PheWAS), Genome-wide association studies (GWAS), genetic interactions, environment-wide association studies (EWAS); gene-environment interactions (GxE), and copy number and rare variant analyses.
```

Comment:
Change to "This app performs genetic associations using PLATO (the PLatform for the Analysis, Translation, and Organization of large-scale data). PLATO is a software tool equipped to handle whole-genome sequence data for hundreds of thousands of samples to explore complexity and genetic architecture underlying common traits using phenome-wide association studies (PheWAS), genome-wide association studies (GWAS), genetic interactions, environment-wide association studies (EWAS); gene-environment interactions (GxE), and copy number and rare variant analyses."